### PR TITLE
fix(types): matching the models like the frontend

### PR DESCRIPTION
**Pull request by nahuelRo**

### CODE REVIEW

File: api/types/order.ts [hunk 1]
```diff
@@ -1,22 +1,22 @@
...
 };
 
-export const OrderStatuses = {
-  confirmed: "Confirmed",
-  canceled: "Canceled",
-  confirmationPending: "Confirmation pending",
-  paymentPending: "Payment pending",
-} as const;
+export const ORDER_STATUSES = [
+  "OrderConfirmed",
```
Comment (sebikap-gm2) [inline api/types/order.ts]: `Confirmed`

File: api/services/member.ts [hunk 2]
```diff
@@ -0,0 +1,56 @@
...
+    return await MemberRepository.create(data);
+  }
+
+  static async update(id: Member["_id"], data: Member) {
+    const MemberUpdated = await MemberRepository.findByIdAndUpdate(id, data, {
+      new: true,
+    });
+
+    if (MemberUpdated) {
+      throw new Error("the team member was not found");
```
Comment (sebikap-gm2) [inline api/services/member.ts:41]:
```
const ERRORS = {
   MemberNotFound: 'The member is not registered',
} as const
type Error = keyof typeof ERRORS
type ErrorMessage = typeof ERRORS[Error]

throw new Error(ERRORS.MemberNotFound)
```

File: api/models/Member.models.ts [hunk 3]
```diff
@@ -67,7 +67,4 @@ const TeamsMemberSchema = new mongoose.Schema({
   },
 });
 
-export const TeamMemberRepository = mongoose.model(
-  "TeamMember",
-  TeamsMemberSchema
-);
+export const MemberRepository = mongoose.model("TeamMember", MemberSchema);
```
Comment (sebikap-gm2) [inline api/models/Member.models.ts]: TeamMember -> Member

**Review by sebikap-gm2 — CHANGES_REQUESTED** [review]

File: api/services/member.ts (hunk 2 above)
Comment (nahuelRo) [reply api/services/member.ts:41]: TODO: in a different branch for all services

**Review by sebikap-gm2 — APPROVED** [review]

**Pull request merged**